.github/workflows/codeberg-mirror.yml

```
@@ -1,25 +1 @@
-    if: github.repository == 'StayRealHQ/Universal'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - uses: Ximaz/repository-mirroring-action@main
-        with:
-          target_repo_url: "git@codeberg.org:StayRealHQ/Universal.git"
-          ssh_private_key: ${{ secrets.CODEBERG_SSH_PRIVATE_KEY }}
-          ssh_private_key_passphrase: ${{ secrets.CODEBERG_SSH_PASSPHRASE }}
+1q_2582_2599_xdelta3.delta","size":942375,"sha3-384":"2fb516f768a171a3f972482b934b12f49657179394d82aba5933014b195e5751183c5ba833f4b28b7204f377ed0840cb"}]},"side-info":{"name":"core20","snap-id":"DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q","revision":"2599","channel":"latest/stable","links":{"issues":["https://github.com/snapcore/core20/issues"],"source":["https://github.com/snapcore/core20"],"website":["https://snapcraft.io"]},"title":"Core 20 base snap","summary":"Snap runtime environment","description":"Base snaps are a specific type of snap that include libraries and dependencies common to many applications. They provide a consistent an
```

actionlint (1.7.7)

workflow is scalar node but mapping node is expected

(syntax-check)